src/components/admin/RightSidebar.tsx

```
@@ -0,0 +1,522 @@
+                        <span className="text-sm text-muted-foreground">(default)</span>
+                    )}
+                </div>
+                <div className="text-sm text-muted-foreground">
+                    No component found for field type: {fieldDefinition.type}
+                </div>
+            </div>
+        );
+    }
+
+    // Memoize cleanField to prevent new object references on every render
+    const cleanField = React.useMemo(() => {
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

Hooks should not be called after an early return.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)